packages/api/scripts/tap-parser.js

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+
+
+/*
+File with utilities for parsing Tap files and outputting them as Csv or more readable format
+*/
+
+
+var fs = require("fs");
+const path = require("path");
+
+
+//class representing single test case
+class TestCaseRun{
```

Comment:
depending on the approach we take, I could imagine we eventually move the tap parser into the `lib/` folder.